keeper.go

```diff
@@ -637,6 +644,28 @@ func (k *keeper) GetHashMember(identifier string, key string) (value interface{}
 	return
 }
 
+// GetMultiHashMembers return type is *goredis.StringCmd
+// to reduce looping and casting, so the caller is the one that should cast it
+func (k *keeper) GetMultiHashMembers(mapKeyToBucket map[string]string) (replies []interface{}, err error) {
```

Comment:
kenapa ga array of struct aja?
kalo dia ada multiple key yang bucketnya sama kan ga bisa pake map

Reply:
oiya, bisa jg, tapi harusnya ga kena timpa sih, soalnya `map[StoryID]: [Bucket]`